modules/geo-layers/src/tile-layer/tile-layer.js

```diff
@@ -53,7 +53,7 @@ export default class TileLayer extends CompositeLayer {
         tileset.finalize();
       }
       tileset = new Tileset2D({
-        getTileData,
+        getTileData: this.state.getTileData || getTileData,
```

Comment:
Alternatively, we can create a `getTileData` method that subclasses can override. By default, it just returns `this.props.getTileData()`

Reply:
Done.  Let me know if it makes sense for you.